scripts/docker/polkadot/polkadot_builder.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM docker.io/paritytech/ci-linux:production as builder
+LABEL io.parity.image.authors="devops-team@parity.io" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.title="builder" \
+	io.parity.image.description="This is the build stage for Polkadot. Here we create the binary." \
+	io.parity.image.source="https://github.com/paritytech/polkadot/blob/master/scripts/docker/polkadot/polkadot_builder.Dockerfile" \
+	io.parity.image.documentation="https://github.com/paritytech/polkadot/scripts/docker/polkadot/README.md"
+
+ARG PROFILE=release
```

Comment:
Not a blocker but I ended up removing this ARG in other images as I don't think it makes much sense to build anything but a release version.

Reply:
Yeah, totally, I think it was there more for explicity.

Reply:
I initially used that when I made the first images thinking `debug` image could be useful but that never happened to be really the case.